src/test/java/core/basesyntax/service/impl/ValidatorTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.storage.DataBase;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ValidatorTest {
```

Comment:
You have only tests when validator throws exception, it's better to add valid case as well

Reply:
Done